Makefile

```diff
@@ -158,12 +159,14 @@ config/.flag.mk: $(CONTROLLER_GEN) $(MANIFEST_GEN_INPUTS)
 	@touch config/.flag.mk
 
 CONVERSION_GEN_TARGET=$(shell find api -type d -name "v*1" -exec echo {}\/zz_generated.conversion.go \;)
-CONVERSION_GEN_INPUTS=$(shell find ./api -name "*test*" -prune -o -name "*zz_generated*" -prune -o -type f -print)
 .PHONY: generate-conversion
-generate-conversion: $(CONVERSION_GEN_TARGET) ## Generate code to convert api/v1beta1 to api/v1beta2
-api/%/zz_generated.conversion.go: bin/conversion-gen $(CONVERSION_GEN_INPUTS)
-	conversion-gen --go-header-file "./hack/boilerplate.go.txt" --input-dirs "./api/v1beta1" \
-	--output-base "." --output-file-base="zz_generated.conversion" --skip-unsafe=true
+generate-conversion: $(CONTROLLER_GEN) ## Generate code to convert api/v1beta1 and api/v1beta2 to api/v1beta3
```

Comment:
```suggestion
generate-conversion: $(CONVERSION_GEN) ## Generate code to convert api/v1beta1 and api/v1beta2 to api/v1beta3
```

Reply:
@hrak **capi-provider-cloudstack-presubmit-e2e-smoke-test** is currently failing because it can't find `conversion-gen`.